docs/user_guide/Troubleshooting/Why_Cant_I_Login.rst

```diff
@@ -123,4 +125,4 @@ If both PAM and DNS appear to be correct, you should next validate that your
 
 See :ref:`pki_validation` for additional guidance.
 
```

Comment:
@lnemsick-simp `nslookup` still works and is always there. I'm not 100% sure that `dig` always shows up.